.tmp/tasks/task-04-refactor-community.md

```
@@ -0,0 +1,158 @@
+| `plugins.community.isFollowed` | WebSocket | `core.js/IsFollowed` | `{cid: number}` |
+| `plugins.community.isCommunityOwner` | WebSocket | `permissions.js/IsCommunityOwner` | `{cid: number}` |
+| `plugins.community.getCommunityData` | WebSocket | `core.js/GetCommunityData` | `{cid: number}` |
+| `plugins.community.updateCommunityData` | WebSocket | `core.js/UpdateCommunityData` | `{cid, name, description, backgroundImage?, icon?, color?, bgColor?}` |
+| `plugins.community.getSubCategories` | WebSocket | `categories.js/GetSubCategories` | `{cid: number}` |
+| `plugins.community.createSubCategory` | WebSocket | `categories.js/CreateSubCategory` | `{parentCid, name, description?, icon?, color?, bgColor?}` |
+| `plugins.community.updateSubCategory` | WebSocket | `categories.js/UpdateSubCategory` | `{cid, parentCid, name, description?, icon?, color?, bgColor?}` |
+| `plugins.community.deleteSubCategory` | WebSocket | `categories.js/DeleteSubCategory` | `{cid: number, parentCid: number}` |
+| `plugins.community.reorderSubCategories` | WebSocket | `categories.js/ReorderSubCategories` | `{parentCid: number, categoryIds: number[]}` |
+| `plugins.community.getMembers` | WebSocket | `members.js/GetMembers` | `{cid: number}` |
+| `plugins.community.addMember` | WebSocket | `members.js/AddMember` | `{cid: number, username: string}` |
+| `plugins.community.changeMemberRole` | WebSocket | `members.js/ChangeMemberRole` | `{cid, targetUid, newRole: 'owner'|'manager'|'member'|'banned'}` |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)